packages/manager/modules/pci/src/components/project/gateways/gateways-model-selector/controller.js

```diff
@@ -1,21 +1,40 @@
 export default class PciProjectGatewayModelSelectorCtrl {
   /* @ngInject */
-  constructor(PciProjectGatewayModelSelectorService) {
+  constructor(coreConfig, PciProjectGatewayModelSelectorService) {
+    this.coreConfig = coreConfig;
     this.PciProjectGatewayModelSelectorService = PciProjectGatewayModelSelectorService;
   }
 
   $onInit() {
     this.loading = true;
+    this.currency = null;
+    this.gateway = {};
+    this.currencySymbol = this.coreConfig.getUser().currency.symbol;
     this.PciProjectGatewayModelSelectorService.getGatewayCatalog(
       this.ovhSubsidiary,
     )
       .then((data) => {
-        this.gatewayCatalog = data;
+        this.gatewayCatalog = [
+          ...new Set(data.map((gatewayInfo) => gatewayInfo.product)),
+        ].map((product) => {
+          return {
+            product,
+            gatewayInfo: data
+              .filter((gateway) => {
+                return gateway.product === product;
+              })
+              .map((gateway) => {
+                return gateway;
+              }),
+          };
+        });
         [this.selectedGatewaySize] = this.gatewayCatalog;
         this.onGatewaySizeSelection({ gateway: this.selectedGatewaySize });
       })
       .finally(() => {
         this.loading = false;
       });
   }
+
+  convertGatewayPrice = (value) => value / 100000000;
```

Comment:
**suggestion:** `100000000` can be maintained as a constant.

Reply:
@anooparveti yes we can but, this I am not going to use anywhere else and creating a constant file just one variable doesn't sounds good to me so I didn't create one.

Reply:
maybe we could use the catalog-price util or something generic instead of having to use another function for conversion

Reply:
Taken care.